ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -97,8 +97,12 @@ public protocol DrawerControllerDelegate: AnyObject {
 
     /// Called when drawer is being dismissed.
     @objc optional func drawerControllerWillDismiss(_ controller: DrawerController)
+
     /// Called after drawer has been dismissed.
     @objc optional func drawerControllerDidDismiss(_ controller: DrawerController)
+
+    /// Called when drawer is getting dismissed when user tries to dismiss drawer by tapping in background, using resizing handle or dragging drawer to bottom. Use this method to turn off drawer dismiss.
```

Comment:
I'd replace "Use this method to turn off drawer dismiss." with "Use this method to prevent the drawer from being dismissed in these scenarios.".

The term "turn off" seems like the drawer is not dismissable at all.

Reply:
Updated docs